src/hdusd/bl_nodes/nodes/converter.py

```diff
@@ -47,7 +47,7 @@ def export(self):
             res = in1.ceil()
         elif op == 'ROUND':
             f = in1.floor()
-            res = (in1 % 1.0 < 0.5).if_else(f, f + 1.0)
+            res = (in1 % 1.0).if_else('>=', 0.5, f + 1.0, f)
```

Comment:
0.5 rounds down to 0, 0.51 rounds up to 1. 

I think you want > not >=

Reply:
No, with in1=0.5 it rounds to 1.
![image](https://user-images.githubusercontent.com/42409851/158329259-dcef2796-31c5-43ec-bee1-a5f3c371967b.png)